chapter2/tests/t20/terraform/base/remote_exec.tf

```diff
@@ -0,0 +1,17 @@
+resource "null_resource" "execute_commands" {
+  for_each = { for idx, ec2_instance in module.wordpress_ec2_instances : idx => ec2_instance }
+
+  provisioner "remote-exec" {
+    connection {
+      type        = "ssh"
+      user        = var.ec2_user
+      private_key = file(local_sensitive_file.ssh_private_key.filename)
+      host        = each.value.public_ip
+    }
+
+    inline = [
+      "echo 'Hello, Camp!'"
```

Comment:
> export the script into the file. load that file via template parse it and inject here.
> 
> Otherwise the future maintenance is hard

@dmitry-mightydevops
Done